targets/unity-v2/source/Entity/ScreenTimeTracker.cs

```diff
@@ -78,25 +80,31 @@ public void ClientSessionStart(EntityModels.EntityKey entityKey, string playFabU
         public void OnApplicationFocus(bool isFocused)
         {
             EventContents eventInfo = new EventContents();
+            DateTime currentUtcDateTime = DateTime.UtcNow;
 
             eventInfo.Name = "client_focus_change";
             eventInfo.EventNamespace = eventNamespace;
             eventInfo.Entity = entityInfo;
 
             double focusStateDuration = 0.0;
 
+            if (initialFocus)
```

Comment:
When do you set initialFocus to false? Also, are there any threading concerns that would make you want to change initialFocus to static?